jsapp/js/components/common/button.tsx

```diff
@@ -63,7 +62,7 @@ interface ButtonProps {
   isFullWidth?: boolean;
   /** Additional class names. */
   classNames?: string[];
-  onClick: Function;
+  onClick: (event: any) => void;
```

Comment:
Any because making it more precise breaks usage. Note that (e: any): void is more precise than Function, which is (any) => any